eliot/_action.py

```diff
@@ -91,8 +95,47 @@ def current(self):
         return stack[-1]
 
 
-_context = _ExecutionContext()
-current_action = _context.current
+class _ECOwner(object):
+    """Owner of the global execution context.
```

Comment:
What does it mean to own that kind of thing?  Some very high level docs, somewhere, about the big(-ish) picture being addressed with this and the other new code would help a lot I think.

Reply:
Reading more, I guess the main idea of `_ECOwner` is that it is a set-at-most-once-with-default container for the execution context object.  It implements the "don't let two different sub-context systems unknowingly trample on each other" concern of the referenced issue.